app/celery/scheduled_tasks.py

```diff
@@ -313,9 +316,41 @@ def populate_monthly_billing():
 @statsd(namespace="tasks")
 def run_letter_jobs():
     job_ids = dao_get_letter_job_ids_by_status(JOB_STATUS_READY_TO_SEND)
-    notify_celery.send_task(
-        name=TaskNames.DVLA_JOBS,
-        args=(job_ids,),
-        queue=QueueNames.PROCESS_FTP
-    )
-    current_app.logger.info("Queued {} ready letter job ids onto {}".format(len(job_ids), QueueNames.PROCESS_FTP))
+    if job_ids:
+        notify_celery.send_task(
+            name=TaskNames.DVLA_JOBS,
+            args=(job_ids,),
+            queue=QueueNames.PROCESS_FTP
+        )
+        current_app.logger.info("Queued {} ready letter job ids onto {}".format(len(job_ids), QueueNames.PROCESS_FTP))
+
+
+@notify_celery.task(name="run-letter-api-notifications")
+@statsd(namespace="tasks")
+def run_letter_api_notifications():
+    current_time = datetime.utcnow().isoformat()
+
+    notifications = dao_set_created_live_letter_api_notifications_to_pending()
```

Comment:
What happens if there is an exception after marking these notifications as pending?
Do we need to build an alert?

Reply:
i naively hope that we have an alert for uncaught exceptions in celery tasks, but will double check with sakis

Reply:
we don't have an alert at the moment but it is straightforward for @sakisv  to add